### DISCUSSION TITLE
Please update recommended harware

### DISCUSSION THREAD

**Discussion by digitalit** [blakeblackshear/frigate#4236] [General]
Hi guys,

I love Frigate and i will use it as long as it's possible.

I bought a bounch of Dahua cameras to be sure it works and i had to buy newer models as they were outgone in Sweden.

I would love if you recommend some cameras that are newer so i can buy them in Sweden.

Thank you and have a nice day.

Best regards
/Peter Carlsson
Sweden

**Comment by yurgh** [comment]
If the camera provides h264 over an RTSP stream it will work great with Frigate. h265/HEVC is not ideal since it needs to be transcoded to be viewed due to browser issues.
